tests/test_interconnect/test_interconnect_cgra.py

```diff
@@ -437,3 +438,143 @@ def test_interconnect_sram(cw_files, add_pd, io_sides):
                                magma_output="coreir-verilog",
                                directory=tempdir,
                                flags=["-Wno-fatal"])
+
+
+@pytest.mark.parametrize("add_pd", [True, False])
+@pytest.mark.parametrize("depth", [1, 10, 100])
```

Comment:
What's the difference between 10 and 100?

Reply:
changed it to 1024 (maximum size for a single tile instead of 100 for corner case)